test/unit/task_tests.rb

```diff
@@ -178,12 +178,16 @@ class RunTests < InitTests
       @call_orders = CallOrders.new
 
       # build a base runner and task manually so the callback tasks actually
-      # run (b/c the test runner doesn't run callbacks)
+      # run (b/c the test runner doesn't run callbacks).  Note: the task's
+      # callbacks are intentionally duplicated on the runner to ensure that
+      # duplicate runner/task callbacks are ignored and the callback is only
+      # run once
```

Comment:
@jcredding add duplicate runner (ie from the config) and task callbacks here to test that any duplicates between the runner/task are ignored.  You cool with this?